src/component/Run.js

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect, useState} from 'react';
+import styled from "styled-components";
+import {useSelector, useDispatch} from 'react-redux';
+
+const Run = () => {
+    // const dataStreamStore = useSelector(state => state.todo.dataStream);
+    // const dateRangeStore = useSelector(state => state.todo.rangeDate);
+
+    const session = useSelector(state => state.session);
+    console.log("1111");
+
+    console.log(session);
+
+    return (
+        <Container>
+            <Header top={-130} left={-180}>
+                Run Page
+            </Header>
+
+            <Position top={-40} left={-165}>
+                Selected data stream: {session.config.dataStream.label}
```

Comment:
here for example - if config or dataStream don't exist we will get an error.
in the past we solved it like so
`let label = '';
if (session.config && session.config.dataStream) {
  label = session.config.dataStream.label;
}`

instead use lodash for that, if something not exist it will return undefined and not break the app - 

import {get} from 'lodash/fp';

get('config.dataStream.label', session)